pl_bolts/datamodules/experience_source.py

```diff
@@ -0,0 +1,201 @@
+"""
+Datamodules for RL models that rely on experiences generated during training
+
+Based on implementations found here: https://github.com/Shmuma/ptan/blob/master/ptan/experience.py
+"""
+from collections import deque
+from typing import Iterable, Callable, Tuple, List
+import numpy as np
+import torch
+from gym import Env
```

Comment:
can we make this import optional? I'm not sure I want gym to be installed, when not doing something related to RL

Reply:
Yes that's a good point, the only reason its here is to have an Env type, but I guess this could be used with other use cases